### DISCUSSION TITLE
I am new here :)

### DISCUSSION THREAD

**Discussion by fredcaixeta** [official-stockfish/Stockfish#4506] [Q&A] [answered]
Hello, my new community!

I'm Fred, and I'm “new” to coding.

I have an engineering background, but I got some C++ (and python) classes (like 2) a couple of years ago.

I still feel like a know here I'm at, knowing how to understand and read codes, but I'm pretty lost at this website and where to start.

I'd love to talk more. Maybe find some friends. I'd love to work on some projects... well, get going.

Cheers

**Comment by dav1312** [comment, marked as answer]
Hello Fred, to have a chat with the Stockfish community you can join the Discord server found in the readme file or in the wiki
